src/views/Viewer.vue

```diff
@@ -24,6 +24,7 @@
 <template>
 	<Modal v-if="initiated || currentFile.modal"
 		id="viewer"
+		size="large"
```

Comment:
This should have been full,  this is a regression
| Before | After |
|:---------:|:------:|
|![2022-04-29_08-43](https://user-images.githubusercontent.com/14975046/165896338-a6d80c6a-2aec-4e93-a96e-dbdbd946e2bb.png)|![2022-04-29_08-43_1](https://user-images.githubusercontent.com/14975046/165896342-a0fd4b11-88e8-407c-9fe9-8d6332b21457.png)|

Reply:
Oh I see. Do you want to create a pr with a fix or shall I?

Reply:
I'm on it, there are lots of issues we missed :)